src/nvim/api/buffer.c

```diff
@@ -636,6 +634,297 @@ ArrayOf(Integer, 2) buffer_get_mark(Buffer buffer, String name, Error *err)
   return rv;
 }
 
+/// Returns mark info at the given position or mark index
+///
+/// @param buffer The buffer handle
+/// @param namespace a identifier returned previously with namespace_create
+/// @param id [row, col] or mark_id of mark
+/// @param[out] err Details of an error that may have occurred
+/// @return [mark_id, row, col]
+ArrayOf(Object) buffer_mark_index(Buffer buffer, Integer namespace, Object id, Error *err)
```

Comment:
beforewith `mark_set`, `mark_xyz`, I can hit tab and a list of all functions that are mark_, but 

now that it's `set_mark` etc that's not possible.